README.md

```diff
@@ -3,3 +3,6 @@
 ## Demo
 [See online](https://libamtrack.github.io/web/)
 
+## Documentation
+[See docs](https://libamtrackweb.readthedocs.io/en/feature-bugfixing-v1/index.html)
```

Comment:
Why are you adding a link to documentation for a branch ?

Reply:
I add link for branch, because it was the only version I created.  I will change it to the latest version after merging - now it's empty -> [link]( https://libamtrackweb.readthedocs.io/en/latest/)

Reply:
ok